pkgs/tools/security/notary/default.nix

```diff
@@ -58,7 +48,7 @@ buildGoPackage rec {
       integrity of the received content.
     '';
     license = licenses.asl20;
-    homepage = "https://github.com/theupdateframework/notary";
+    homepage = "https://github.com/notaryproject/notary";
     maintainers = with maintainers; [ vdemeester ];
     platforms = platforms.unix;
```

Comment:
```suggestion
```

Probably best to leave this to `buildGoModule` unless it's narrower than `buildGoModule`